scripts/language_model/word_language_model.py

```diff
@@ -151,18 +151,21 @@
 # Build the model
 ###############################################################################
 
-
 ntokens = len(vocab)
 
 if args.weight_dropout > 0:
     print('Use AWDRNN')
-    model = nlp.model.language_model.AWDRNN(args.model, len(vocab), args.emsize,
-                                            args.nhid, args.nlayers, args.tied,
-                                            args.dropout, args.weight_dropout, args.dropout_h,
-                                            args.dropout_i, args.dropout_e)
+    model = nlp.model.train.AWDRNN(args.model, len(vocab), args.emsize, args.nhid, args.nlayers,
+                                   args.tied, args.dropout, args.weight_dropout,
+                                   args.dropout_h, args.dropout_i, args.dropout_e)
+    model_eval = nlp.model.AWDRNN(args.model, len(vocab), args.emsize, args.nhid, args.nlayers,
+                                  args.tied, args.dropout, args.weight_dropout,
+                                  args.dropout_h, args.dropout_i, args.dropout_e)
```

Comment:
since there is inheritance, only one instance is enough. using `super`, you can create an instance that behaves like parent class but share all the states with child class. for example:
```python
In [1]: class A(object):
   ...:     def __init__(self, v):
   ...:         self.v = v
   ...:     def get(self):
   ...:         return 'A' + str(self.v)
   ...:

In [2]: class B(A):
   ...:     def get(self):
   ...:         return 'B' + str(self.v)
   ...:

In [3]: b = B(123)

In [4]: a = super(B, b)

In [5]: b.get()
Out[5]: 'B123'

In [6]: a.get()
Out[6]: 'A123'
```

Reply:
made the changes.